### PR TITLE
SmallerNoArray

### PR DESCRIPTION
**Pull request by namrata124**

### CODE REVIEW

File: leetcode/src/main/java/com/leet/SmallerNoArray.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        int count = 0;
+
+        for (int i = 0; i < nums.length; i++) {
+            for (int j = 0; j < nums.length; j++) {
+
+                if (j != i && nums[j] < nums[i]) {
+
+                    count++;
+                }
+
```
Comment (usrivastava92) [inline leetcode/src/main/java/com/leet/SmallerNoArray.java:19]: why there are 2-3 lines between 2 lines of code. Try to write clean and formatted code.

File: leetcode/src/main/java/com/leet/SmallerNoArray.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        return returnNums;
+
+    }
+
+    public static void main(String[] args) {
+        int[] nums = {2, 5, 6, 8};
+        int[] output = smallerNumbersThanCurrent(nums);
+
+        for (int i = 0; i < output.length; i++) {
```
Comment (usrivastava92) [inline leetcode/src/main/java/com/leet/SmallerNoArray.java:37]: what is the purpose of this loop here?

File: leetcode/src/main/java/com/leet/SmallerNoArray.java (hunk 1 above)
Comment (namrata124) [reply leetcode/src/main/java/com/leet/SmallerNoArray.java:19]: ok

File: leetcode/src/main/java/com/leet/SmallerNoArray.java (hunk 2 above)
Comment (namrata124) [reply leetcode/src/main/java/com/leet/SmallerNoArray.java:37]: itrate tand print he output array

Comment (usrivastava92) [reply leetcode/src/main/java/com/leet/SmallerNoArray.java:37]: manual logic for this is not necessary, instead you can use System.out.println(Arrays.toString(output));

Comment (namrata124) [reply leetcode/src/main/java/com/leet/SmallerNoArray.java:37]: not knowing this

**Review by usrivastava92 — APPROVED** [review]

**Pull request merged**